pkg/daemon/server/handle.go

```diff
@@ -121,11 +121,26 @@ func (cdh *cniDaemonHandler) handleAdd(req *restful.Request, resp *restful.Respo
 			return
 		}
 
-		if (len(ipInstanceList) == 1 && (ipFamily == ipamtypes.IPv4Only || ipFamily == ipamtypes.IPv6Only)) ||
-			(len(ipInstanceList) == 2 && ipFamily == ipamtypes.DualStack) {
+		var expectIPNumber int
+		switch ipFamily {
+		case ipamtypes.IPv4Only:
```

Comment:
case support multiple values in one line, is it better to `case IPv4Only, IPv6Only` to save several lines?

Reply:
Of course, it's done.